src/utils/index.ts

```diff
@@ -4,3 +4,4 @@ export {KeyMap, KeyCode} from './key-map';
 export {default as getLogger} from './logger';
 export {withKeyboardA11y} from './popup-keyboard-accessibility';
 export * from './time-format';
+export {getTimeInText} from './duration-humanizer';
```

Comment:
please also export it on the ui so we can use it from other places. see example here- https://github.com/kaltura/playkit-js-ui/blob/master/src/index.ts#L20, https://github.com/kaltura/playkit-js-ui/blob/master/src/index.ts#L24

Reply:
done